conans/test/integration/toolchains/gnu/test_pkgconfigdeps.py

```diff
@@ -431,11 +441,25 @@ def package_info(self):
     assert content == pc_content
 
     pc_content = client.load("pkg_other_name.pc")
-    assert "Description: Conan package: pkg_other_name" in pc_content
-    assert "Requires: compo1" in pc_content
+    content = textwrap.dedent(f"""\
+    {prefix}
+    # Custom PC content
+
+    datadir=${{prefix}}/share
+    schemasdir=${{datadir}}/mylib/schemas
```

Comment:
Added this extra check to ensure that the custom content is being added only to the root .pc file.